composer.json

```diff
@@ -19,12 +19,18 @@
     "config": {
         "sort-packages": true
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/webimpress/zend-expressive-router.git"
```

Comment:
Why is this package not published on packagist.org?

Reply:
@zf2timo This PR requires another one to be merged before - https://github.com/zendframework/zend-expressive-router/pull/36
After merge PR in expressive-router, this PR will be updated.